.gitignore

```diff
@@ -32,3 +32,11 @@ webroot/forum/files/*
 webroot/forum/images/avatars/gallery/*
 webroot/forum/images/avatars/upload/*
 webroot/forum/store/*
+
```

Comment:
This entire section can be outsourced to your personal (i.e. local) gitignore as designated by Philippe: https://github.com/thewca/worldcubeassociation.org/pull/6312

Reply:
Woah - had no idea this was a thing!